cadc-download-manager-server/src/main/resources/META-INF/resources/chooser.jsp

```diff
@@ -119,14 +121,16 @@ String bodyFooter = skin + "bodyFooter";
         <div style="padding-left: 2em; padding-right: 2em">
         <table width="66%">
             <tbody>
-                <tr>
-                    <td valign="top"><input type="submit" name="method" value="<%= DispatcherServlet.WEBSTART %>" /></td>
-                    <td valign="top">
-                    
-                    <jsp:include page='javaWebStartDescription.html' flush='true' />
-                    
-                    </td>
-                </tr>
+                <c:if test="${enableWebstart eq 'true'}" >
```

Comment:
It's a boolean.  Will it ever be equal to 'true'?  It should be consistent with the `<c:if test="<%=enableWebstart%>" >` below.